src/HTTP/HTTPServer.cpp

```diff
@@ -163,7 +163,7 @@ bool cHTTPServer::Start(cCallbacks & a_Callbacks, const AStringVector & a_Ports)
 		}
 		reportPorts.append(port);
 	}
-	LOGINFO("WebAdmin is running on port(s) %s", reportPorts.c_str());
+	LOGINFO("HTTPServer is running on port(s) %s", reportPorts.c_str());
```

Comment:
It would be a good idea to identify WebAdmin a bit better in the logs than just HTTPServer - maybe this is another area that needs refactoring.

Reply:
The problem is that this code is in `HTTPServer.cpp `.
Even if this was not confusing because the only HTTPServer was WebAdmin, this is not really clean.